src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -301,5 +301,543 @@ public override void LoadXml(XmlElement element)
                 AddCertificate(new X509Certificate2(Convert.FromBase64String(Utils.DiscardWhiteSpaces(node.InnerText))));
             }
         }
+
+        //
+        // This is a pretty "crude" implementation of BigInt arithmetic operations.
+        // This class is used in particular to convert certificate serial numbers from
+        // hexadecimal representation to decimal format and vice versa.
+        //
+        // We are not very concerned about the perf characterestics of this implementation
+        // for now. We perform all operations up to 128 bytes (which is enough for the current
+        // purposes although this constant can be increased). Any overflow bytes are going to be lost.
+        //
+        // A BigInt is represented as a little endian byte array of size 128 bytes. All
+        // arithmetic operations are performed in base 0x100 (256). The algorithms used
+        // are simply the common primary school techniques for doing operations in base 10.
+        //
+
+        internal sealed class BigInt
```

Comment:
Hmm, I'm having a bit of difficulty referencing `System.Runtime.Numerics`.

I looked at how the other projects reference it, and so I modified `project.json`:

```json
{
  "frameworks": {
    "netstandard1.7": {
        "dependencies": {
            "NetStandard.Library2": "2.0.0-beta-24814-01",
            "System.Runtime.Numerics": "4.4.0-beta-24814-01"
        }
    },
    "net462": {
      "dependencies": {
        "Microsoft.TargetingPack.NETFramework.v4.6.2": "1.0.1",
        "System.Runtime.Numerics": "4.3.0"
      }
    }
  }
}
```

But as soon as I do that, restore packages, and build, I get:

```
1>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3243: No way to resolve conflict between "System.Runtime.Numerics, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Runtime.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". Choosing "System.Runtime.Numerics, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" arbitrarily.
1>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3243: No way to resolve conflict between "System.Runtime, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". Choosing "System.Runtime, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" arbitrarily.
```

I might put this one aside for now and come back to it; the main difficulty is that I'm not sure which version this package is compatible.